pkgs/development/python-modules/cppe/default.nix

```diff
@@ -31,7 +33,12 @@ buildPythonPackage rec {
 
   dontUseCmakeConfigure = true;
 
-  buildInputs = [ pybind11 ];
+  buildInputs = [ pybind11 ]
+    ++ lib.optional stdenv.cc.isClang llvmPackages.openmp;
+
+  NIX_CFLAGS_LINK = lib.optional stdenv.cc.isClang "-lomp";
```

Comment:
What's the difference between NIX_CLFAGS_LINK  and NIX_LDFLAGS ?

Reply:
That is such a good question.